README.md

```diff
@@ -736,6 +742,154 @@ No errors detected
 
 ```
 
+## `mcci-catena-provision-sigfox.py`
+
+This script communicates with catena to get information for register it in sigfox network via sigfox API, then it loads the catena script to device for complete the provisioning process.
+
+### Required
+
+* Python 3.* (Installation steps [here](https://realpython.com/installing-python/))

```

Comment:
We need a comment (if it's not above) that we recommend Real Python on Windows and (what? on other systems:  choices are  "Python 3.5 or greater", "we haven't tested, but it might work", or "good luck to you" 😊 -- but we should mention explicitly the situation for other platforms than Windows).